CHANGELOG.rst

```diff
@@ -1,6 +1,10 @@
 Unreleased_
 -----------
 
+Added
+^^^^^
+* `@senwu`_: Add test for unary relation featurization.
+
```

Comment:
This isn't user-facing changes, so no need to change the changelog.

Reply:
Removed.